gui/wxpython/datacatalog/tree.py

```diff
@@ -477,10 +478,12 @@ def OnRightClick(self, node):
 
         if self.selected_layer[0]:
             self._popupMenuLayer()
-        elif self.selected_mapset[0] and not self.selected_type[0] and len(self.selected_mapset) == 1:
-            self._popupMenuMapset()
         elif self.selected_type[0] and len(self.selected_type) == 1:
             self._popupMenuElement()
+        elif self.selected_mapset[0] and not self.selected_type[0] and len(self.selected_mapset) == 1:
+            self._popupMenuMapset()
+        elif self.selected_location[0] and not self.selected_type[0] and len(self.selected_location) == 1:
```

Comment:
not sure why the selected_type, I think it should be selected_mapset instead, but I am not sure, please test that.